src/daria/analysis/segmentationcomparison.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import annotations
+
+import numpy as np
+
+
+def compare_segmentations(*segmentations, **kwargs) -> np.ndarray:
+    """
+    Comparison of segmentations.
+
+    Args:
+        segmentations (asterisk argument): Allows to provide an arbitraty
+                number of segmented images to be compared
```

Comment:
I would rather say numpy arrays instead of images, jpg is not allowed e.g. Also mention that these arrays are expected to be filled with int values.